datafusion-cli/src/helper.rs

```diff
@@ -166,40 +154,8 @@ impl Validator for CliHelper {
 
 impl Helper for CliHelper {}
 
-/// Unescape input string from readline.
-///
-/// The data read from stdio will be escaped, so we need to unescape the input before executing the input
-pub fn unescape_input(input: &str) -> datafusion::error::Result<String> {
```

Comment:
So this function was the source of the bug. Per the comment above, someone had a reason to write it though. Do you maybe know what changed? Like, did we change how we obtain input lines?

Reply:
One thing that might have changed is that sqlparser got much more sophistcated in handling escaping itself / added a bunch of handling for the escaping in the parser (like maybe \u{} style syntax 🤔 )